api/Request.json

```diff
@@ -772,7 +816,7 @@
                 "version_added": "68"
               },
               "chrome_android": {
-                "version_added": "68"
+                "version_added": false
```

Comment:
I don't know where 68 came from but it actually [shipped in 72](https://storage.googleapis.com/chromium-find-releases-static/ddb.html#ddb23f27ff0cdcc155f9eb535ad03ab0cd04b3f4) in Chrome, Android, and Webview. 

What made you think it hadn't shipped in Android?

Reply:
Apparently someone added that its parent, `credentials`, was removed from Chrome Android in version 46...which obviously doesn't make any sense.  Fixed!